app/pages/sections.py

```
@@ -0,0 +1,826 @@
+  </div>
+</section>
+"""
+
+
+# ---------------------------------------------------------------------------
+# Developer panel  (the original root.py content, preserved as a section)
+# ---------------------------------------------------------------------------
+
+def developer_panel(app_name: str) -> str:
+    """Quickstart, health, and route reference for API consumers."""
+    quickstart_body = f"""
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)